### PR TITLE
images/openwrt: Use apk as package manager

### PR DESCRIPTION
**Pull request by MusicDin**
Recently, openWRT changed their package manager from "opkg" to "apk" (Alpine Linux package manager).

Post from openWRT forum: https://forum.openwrt.org/t/the-future-is-now-opkg-vs-apk/201164

Passing build: https://github.com/MusicDin/lxd-ci/actions/runs/11940145394/job/33282183927

### CODE REVIEW

**Comment by MusicDin** [comment]
@simondeziel I've noticed that on this PR https://github.com/canonical/lxd-ci/pull/358 the tests were not run, but on this one all the tests are executed. Should we also disable them if changes are made to `.github/workflows/images_*.yaml`?

File: images/openwrt.yaml [hunk 1]
```diff
@@ -206,13 +206,16 @@ files:
...
+  # Note: Older releases of OpenWrt use "opkg" package manager.
+  manager: apk
   update: false
   cleanup: true
   sets:
   - packages:
     - sudo
     action: install
+    releases:
+    - 23.05
```
[the remark this answers is not in the document]
Comment (simondeziel) [reply images/openwrt.yaml:218]: Then why is this setting `apkg` a few lines before?

[the remark this answers is not in the document]
Comment (MusicDin) [reply images/openwrt.yaml:218]: From now on it will be `apk` - so 23.05 is last release to use opkg.

[the remark this answers is not in the document]
Comment (simondeziel) [reply images/openwrt.yaml:218]: With that change, is it correct that `sudo` is no longer installed on `snapshot`? Is it the distro that moved to `doas` already?

**Review by simondeziel — APPROVED** [review]

File: images/openwrt.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (MusicDin) [reply images/openwrt.yaml:218]: Ahh, sorry. Yes you are right. I was so focused on the package manager selection, that haven't realized your point.

[the remark this answers is not in the document]
Comment (MusicDin) [reply images/openwrt.yaml:218]: https://github.com/canonical/lxd-ci/pull/360

**Pull request merged**